tests/shadow-dom.stories.tsx

```diff
@@ -47,3 +47,72 @@ export const Nested = () => (
     <KeyboardMode />
   </div>
 );
+
+interface WindowWithCreateKeyborg extends Window {
+  createKeyborg?: typeof createKeyborg;
+}
+
+export const Lazy = () => {
+  (window as WindowWithCreateKeyborg).createKeyborg = createKeyborg;
```

Comment:
instead of using an eval to create keyborg from a global instance, add a button that creates an instance of keyborg on click

Reply:
Likewise the event listeners can be added on the example side and output to the DOM.

Reply:
I'll do a follow-up on how the tests are written.